File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/dart/practicas_3/edad.dart
import 'dart:io';

void main () {

  int old = 0;
  int young = 0;

  print('¿Cuantas personas se van a encuestar sobre las edades?');
  int cuantos = int.parse(stdin.readLineSync()!);

  for (int vueltas = 1; vueltas <= cuantos; vueltas++) {

    print('----------------------------------------------------------');

    print('¿Cuantos años tiene?');
    int age = int.parse(stdin.readLineSync()!);

    if (age >= 18){
      print('Usted es mayor de edad');
      old = old + 1;
    } else {
      print('Usted no es mayor de edad');
      young = young + 1;
    }

    print('----------------------------------------------------------');

  }

  print('----------------------------------------------------------');

  print('RESULTADOS DE LA ENCUESTA:');

  print('----------------------------------------------------------');

  print('Personas mayor de edad:');
  print(old);

  print('Personas menor de edad:');
  print(young);

  print('----------------------------------------------------------');


}

File: keiner_lindarte_aro/instructor_henry/mis_programas/practicas/ruby/practicas_3/edad.rb
old = 0
young = 0

puts "¿Cuantas personas se van a encuestar sobre las edades?"
cuantos = gets.chomp.to_i

for vueltas in 1..cuantos

    puts "----------------------------------------------------------"

    puts "¿Cuantos años tiene?"
    age = gets.chomp.to_i

    if age >= 18
      puts "Usted es mayor de edad"
      old = old + 1
    else
      puts "Usted no es mayor de edad"
      young = young + 1
    end

    puts "----------------------------------------------------------"

end

puts "----------------------------------------------------------"

puts "RESULTADOS DE LA ENCUESTA:"

puts "----------------------------------------------------------"

puts "Personas mayor de edad:"
puts old

puts "Personas menor de edad:"
puts young

puts "----------------------------------------------------------"